app/src/main/java/com/zendrive/zendrivesdkdemo/MainFragment.java

```diff
@@ -2,14 +2,12 @@
 
```

Comment:
General comment: Should we teardown Zendrive SDK when all activities are killed?